go/vt/vttest/environment.go

```diff
@@ -155,6 +163,15 @@ func (env *LocalTestEnv) MySQLManager(mycnf []string, snapshot string) (MySQLMan
 	}, nil
 }
 
+func (env *LocalTestEnv) IsSQLFlavor() bool {
+	for _, val := range env.Env {
+		if strings.Contains(val, "MYSQL_FLAVOR=MySQL") {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
We already have a flavor package. I don't see why we need this. Even more so because using the env var is only one way to set this.